GameData/SPEngine/Parts/Agena_proc.cfg

```diff
@@ -11,3 +11,17 @@
 	!identicalParts = // we're not identical to the part we copied!
 	!engineType = // prevent ROE from deleting us
 }
+
++PART[ROE-Agena8048]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
+{
+	@name = SPEngineAgena
+	@title = G-class Procedural Engine
+	@description = Pump-fed hypergolic vacuum engine, like Agena (Bell XLR81).  <b><color=green>From Ven Stock Revamp mod</color></b>
+	MODULE
+	{
+		name = ModuleSPEngine
+		familyLetter = G
+	}
+	!identicalParts = // we're not identical to the part we copied!
+	!engineType = // prevent ROE from deleting us
+}
```

Comment:
Again missing newline at end of file.  (Same applies to several more changes below.)